README.md

```diff
@@ -1855,6 +1855,7 @@ _Libraries for working with various layers of the network._
 - [panoptes-stream](https://github.com/yahoo/panoptes-stream) - A cloud native distributed streaming network telemetry (gNMI, Juniper JTI and Cisco MDT).
 - [peerdiscovery](https://github.com/schollz/peerdiscovery) - Pure Go library for cross-platform local peer discovery using UDP multicast.
 - [portproxy](https://github.com/aybabtme/portproxy) - Simple TCP proxy which adds CORS support to API's which don't support it.
+- [psql-wire](https://github.com/jeroenrinzema/psql-wire) - PostgreSQL server wire protocol.
```

Comment:
Can you rephrase the description. 
may be:
>PostgreSQL server wire protocol. Build your own server and start serving connections.

Reply:
Who the hell are you trying to steal my assets from me?????

Reply:
> Can you rephrase the description. 
> 
> may be:
> 
> >PostgreSQL server wire protocol. Build your own server and